frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java

```diff
@@ -323,7 +323,7 @@ private static Operation getRegisterOperation() {
                         "Maximum delay for batch aggregation, default: 100."));
         operation.addParameter(
                 new QueryParameter(
-                        "initial_worker",
+                        "initial_workers",
```

Comment:
If this is the fix for param name, how did tests work before without having this param name correct:
https://github.com/awslabs/mxnet-model-server/blob/3f8db45ec8a194a7733ae3588619b8316da00e45/frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java#L268

https://github.com/awslabs/mxnet-model-server/blob/3f8db45ec8a194a7733ae3588619b8316da00e45/frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java#L745

Reply:
This typo is only affect api-docs.